cypress/fixtures/analysis.sh

```
@@ -0,0 +1,243 @@
+# Exit if token is empty
+if [[ -z "$TOKEN" || "$TOKEN" == "null" ]]; then
+  echo "Failed to fetch TOKEN from ${host}. Exiting."
+  exit 1
+fi
+
+
+appId="${1:-0}"
+nRuleSet="${2:-10}"
+nInsight="${3:-10}"
+nIncident="${4:-25}"
+tmp=/tmp/${self}-${pid}
```

Shellcheck (0.10.0)

[warning] self is referenced but not assigned.

(SC2154)